release/models/wifi/openconfig-wifi-phy.yang

```diff
@@ -25,7 +25,14 @@ module openconfig-wifi-phy {
   description
     "Model for managing PHY layer configuration of Radio interfaces.";
 
-  oc-ext:openconfig-version "1.2.4";
+  oc-ext:openconfig-version "1.2.5";
```

Comment:
should increment the minor version,  ie:  1.3.0

Reply:
Thank you, fixed minor version.